zk_toolbox/crates/zk_supervisor/src/messages.rs

```diff
@@ -113,6 +114,12 @@ pub(super) const MSG_REVERT_TEST_RUN_SUCCESS: &str = "Revert and restart test ra
 pub(super) const MSG_RECOVERY_TEST_RUN_INFO: &str = "Running recovery test";
 pub(super) const MSG_RECOVERY_TEST_RUN_SUCCESS: &str = "Recovery test ran successfully";
 
+// Upgrade tests related messages
+pub(super) const MSG_UPGRADE_TEST_RUN_INFO: &str = "Running upgrade test";
+pub(super) const MSG_UPGRADE_TEST_INSTALLING_DEPENDENCIES: &str =
+    "Building and installing dependencies. This process may take a lot of time...";
+pub(super) const MSG_UPGRADE_TEST_RUN_SUCCESS: &str = "Revert and restart test ran successfully";
```

Comment:
same

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2515/commits/78799acf728d49383e25de636499e6948d57f72b